local-proxy/src/server.ts

```diff
@@ -102,14 +108,18 @@ app.post('/invoke', json(), async (req, res) => {
     // Not logging args to avoid sensitive info log (?)
     reportInvocationDurationUs(startHrtime, requestId, { path, handler });
     if (ret === undefined) {
-      return res.sendStatus(204);
+      ctx.status = 204;
+      return;
     }
-    res.json(ret);
+    // ret might be a scalar, which Koa does not auto-JSONify.  So
```

Comment:
Nice catch.

Reply:
The unit test returns a scalar value (as it should, those are always the hardest).